### PR TITLE
Connor wang/hw1

### PR DESCRIPTION
**Pull request by connorwang0901**

### CODE REVIEW

**Comment by fzybb** [comment]
Well done.

File: Repo/MavenProject/maven/src/main/resources/archetype-resources/src/test/java/AppTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    /**
+     * Rigourous Test :-)
+     */
+    public void testApp()
+    {
+        assertTrue( true );
+    }
+}
```
Comment (zhaohanyun) [inline Repo/MavenProject/maven/src/main/resources/archetype-resources/src/test/java/AppTest.java:38]: that's a nice template

**Comment by ShuaiZhao418** [comment]
Good Good!

**Comment by Frank-Z-Chen** [comment]
LGTM

**Pull request merged**